test/idp/src/main/kotlin/org/codice/compliance/web/sso/PostLoginTest.kt

```diff
@@ -123,8 +124,30 @@ class PostLoginTest : StringSpec() {
             SingleSignOnProfileVerifier(responseDom, acsUrl[HTTP_POST]).verify()
         }
 
+        "POST AuthnRequest With Relay State Greater Than 80 Bytes Test" {
+            Log.debugWithSupplier { "POST AuthnRequest With Relay State Greater Than 80 Bytes Test" }
+            val encodedRequest = Encoder.encodePostMessage(
+                    createValidAuthnRequest(), TestCommon.RELAY_STATE_GREATER_THAN_80_BYTES)
+            val response = given()
+                    .urlEncodingEnabled(false)
+                    .body(encodedRequest)
+                    .contentType("application/x-www-form-urlencoded")
+                    .log()
+                    .ifValidationFails()
+                    .`when`()
+                    .post(Common.getSingleSignOnLocation(POST_BINDING))
+
+            val idpResponse = TestCommon.parseErrorResponse(response)
+
+            idpResponse.bindingVerifier().verifyError(TestCommon.IDP_ERROR_RESPONSE_REMINDER_MESSAGE)
```

Comment:
❗️ I **really** don't like all the custom logic to account for this message being passed down the stack. It's pretty confusing. I also don't feel this is the right place for that message to occur. In my opinon, this message should be hardcoded into the array of messages that `CoreVerifier.verifyErrorStatusCode()` throws and all the parameters for taking an "extra message" here should be removed.

Reply:
Discussed and changing!